package/crds/mq.aws.crossplane.io_brokers.yaml

```diff
@@ -240,6 +240,12 @@ spec:
                       type: object
                     type: array
                 required:
+                - autoMinorVersionUpgrade
```

Comment:
This is potentially a breaking change. Was it possible to create a Broker without required information here? i.e. will we see users that deployed `Broker` without those fields and get error once they upgrade?

Reply:
tested with master and this PR - without issues

```
(⎈ |kind-pr-1122:default)➜  provider-aws git:(master) kubectl apply -f package/crds
[...]
(⎈ |kind-pr-1122:default)➜  provider-aws git:(master) kubectl get server.transfer.aws.crossplane.io 
NAME      READY   SYNCED   EXTERNAL-NAME
example   True    True     s-3debea30e5694c34a

```


```
(⎈ |kind-pr-1122:default)➜  provider-aws git:(feature/ack-bump-sdk-1.42.0) kubectl apply -f package/crds
[...]
(⎈ |kind-pr-1122:default)➜  provider-aws git:(feature/ack-bump-sdk-1.42.0) kubectl get server.transfer.aws.crossplane.io
NAME      READY   SYNCED   EXTERNAL-NAME
example   True    True     s-3debea30e5694c34a

```

Reply:
in general with the old SDK a lot of fields are required without of configurations in sdk

spec.versions.v1alpha1.schema.openAPIV3Schema.properties.spec.properties.forProvider.required
  + six list entries added:
    - autoMinorVersionUpgrade 
    - deploymentMode (x)
    - engineType (x)
    - engineVersion (x)
    - hostInstanceType (x)
    - publiclyAccessible

Reply:
added LateInit https://github.com/crossplane/provider-aws/pull/1122/commits/a19e007229c5238c5c819d0b6424ca7bd29e0d02
it's safe to update the MQ Broker to new provider-aws version

```
(⎈ |kind-pr-1122:default)➜  provider-aws git:(master) ✗ kubectl get managed
NAME                                           READY   SYNCED   EXTERNAL-NAME
broker.mq.aws.crossplane.io/example-activemq   True    True     b-719769f7-96eb-4003-9668-d8c1212d10e8
```

```
(⎈ |kind-pr-1122:default)➜  provider-aws git:(feature/ack-bump-sdk-1.42.0) ✗ kubectl get managed
NAME                                           READY   SYNCED   EXTERNAL-NAME
broker.mq.aws.crossplane.io/example-activemq   True    True     b-719769f7-96eb-4003-9668-d8c1212d10e8
```

```
kubectl get broker.mq.aws.crossplane.io/example-activemq -o yaml  | grep -E "autoMinor|publiclyAccessible"
    autoMinorVersionUpgrade: true
    publiclyAccessible: false
```